core/sr-api-macros/benches/bench.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Parity Technologies (UK) Ltd.
+// Copyright 2019 Parity Technologies (UK) Ltd.
```

Comment:
should be all years that it changed; i.e. `2018, 2019`
```suggestion
// Copyright 2018, 2019 Parity Technologies (UK) Ltd.
```